src/ONYXKEYS.ts

```diff
@@ -1079,14 +1082,20 @@ type OnyxValuesMapping = {
     [ONYXKEYS.LAST_FULL_RECONNECT_TIME]: string;
     [ONYXKEYS.TRAVEL_PROVISIONING]: OnyxTypes.TravelProvisioning;
 };
-type OnyxValues = OnyxValuesMapping & OnyxCollectionValuesMapping & OnyxFormValuesMapping & OnyxFormDraftValuesMapping;
+
+type OnyxDerivedValuesMapping = {
```

Comment:
It would be neat to infer this type out of `ONYX_DERIVED_VALUES`, I think it should be possible 😄 (I could look into it)

Reply:
I was thinking that this would be the source of truth that the implementation (`ONYX_DERIVED_VALUES`) would have to follow.

Reply:
I agree this will be safer in a longer run 👍